app/src/main/res/values/dimens.xml

```diff
@@ -467,6 +467,11 @@
   <dimen name="topic_practice_footer_view_margin_top">36dp</dimen>
   <dimen name="topic_practice_footer_view_margin_bottom">108dp</dimen>
 
+  <!-- Profile Progress Fragment -->
+  <dimen name="profile_progress_fragment_marginStart">20dp</dimen>
```

Comment:
Rename to `profile_progress_fragment_margin_start `
here and everywhere else. All names should be `small_case` separated by `_`

Reply:
Done.